data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -137,11 +112,11 @@ def test_get_output_file_path(self, mock_etl, mock_paths):
         )
         assert output_file_path == expected_output_file_path
 
-    def test_extract(self, mock_etl, mock_paths):
-        """Ensure the extract results are working as appropriate."""
+    def test_extract_path(self, mock_etl, mock_paths):
+        """Ensure the extract is happening where appropriate."""
+        # this is returning true for no reason?
```

Comment:
This might have been what you meant by your comment in the code, but do we still need `test_extract_path()` now that you've implemented `test_extract()`?

Reply:
That is what I meant! but on reflection, I *think* we actually still need this because we are testing the "native" path if that makes sense?